caffe2/operators/abs_op.h

```diff
@@ -0,0 +1,34 @@
+#ifndef CAFFE2_OPERATORS_ABS_OP_H_
+#define CAFFE2_OPERATORS_ABS_OP_H_
+
+#include <vector>
+
+#include "caffe2/operators/elementwise_ops.h"
+#include "caffe2/utils/math.h"
+
+namespace caffe2 {
+
+template <class Context>
+struct AbsFunctor {
+  template <typename T>
+  bool operator()(const int N, const T* X, T* Y, Context* context) const {
+    math::Abs(N, X, Y, context);
+    return true;
+  }
+};
+
+template <class Context>
+struct AbsGradientFunctor {
+  template <typename T>
+  bool Forward(
```

Comment:
call this `compute`? `Forward` sounds a little bit ambiguous to me.

Reply:
Here the gradjent ops for the unary ops are binary ops. So we use the ordinary binary op to define the GradientFunctor. For the binary ops, we need the functor support both forward and backward. Here we only use the forward pass.

Reply:
I know your point. My point is that Unary gradient ops only have one method, calling its method `Compute` should be easy to understand. But calling it `Forward` is also fine, up to you. :-)